OneCommunityGlobal/HighestGoodNetworkApp
# Kaikanes drag and drop remake branch

# Description
Added Drag and Drop feature to my lesson from design
turns out file inputs on mobile automatically have take photo options
(remake with git attribute changes)

## Related PRS (if any):
Added feature to Front end PR
https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1477

## Main changes explained:
- added the ability to drag and drop

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as anyone
4. go to  http://localhost:3000/bmdashboard/lessonform
5. verify drag and drop functionality works in various view ports and scenarios
6. verify take photo option
7. go to https://hgnapplication_react_beta.surge.sh/bmdashboard/lessonform on your phone/tablet
8. Now when you click on the file drag and drop area you should see options to Take Photo or video/ photo library etc.

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/114305309/009342a7-9157-466f-9d28-0b1bfba1ca0f

![3y0ynczp](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/114305309/1425e48a-6efd-4d28-a564-9c4dd789f7c1)

## Note:
This is a remake branch without gitattributes errors
Old branch
https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1502
